src/components/fonts.css

```
@@ -41,21 +41,21 @@
 }
 
 @font-face {
-  font-family: Inter;
+  font-family: 'Inter';
```

Stylelint (17.13.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)